conf/application.conf

```diff
@@ -157,7 +155,7 @@ microservice {
             host = localhost
             port = 2003
             prefix = play.${appName}.
-            enabled = true
+            enabled = false
```

Comment:
why is the session timeout enabled changed to false?

Reply:
It was for my local testing, will change it back

Reply:
PR comments addressed